docs/projects/t2-2022/team.md

```diff
@@ -0,0 +1,151 @@
+# Teams
+
+ ## Technical Learning group
+
+ - Area Lead: [to be added]
+
+ ## Quality Assurance group
+
+ - Area Lead: [to be added]
+
+ ## People & Documentation group
+
+ - Area Lead: [to be added]
+
+ ## Data Science & Analytics group
+
+ - Area Lead: [to be added]
+
+ ## OnTrack group
+
+ - Product Lead: [to be added]
+
+ ### User Interface Enhancement
+
+ - Teams Channel: [User Interface Enhancement](https://teams.microsoft.com/_?tenantId=d02378ec-1688-46d5-8540-1c28b5f470f6#/school/conversations/Front%20End%20Migration?groupId=0e15669c-3f66-49aa-b023-640fe1dda2e0&threadId=19:40cc53f7f52d42cd8d15bddad593aa01@thread.tacv2&ctx=channel)
+ - Trello Board: [User Interface Enhancement](https://trello.com/b/pFPgCaIo/front-end-migration)
+ - Delivery Lead: Leo Luong
+ - Academic Mentor: Muna Al-Hawawreh
+ - Team Members: Nelson Lai, Ahmed Ali, Aldy Putra Shavira, Anthony Papoutsis, Tushar, Haris Sajjad, Kosta Constantinou
```

Comment:
I adjusted some content here already. @Ishrat420 could you double check for us again if everything is good to merge

Reply:
Everything else look fine, just run 'npm run format:check' so that it will pass the check

Reply:
![Screen Shot 2022-08-04 at 2 16 51 am](https://user-images.githubusercontent.com/65574142/182658623-0fb22585-335c-48e8-a829-e4247e54afbd.png)
When I try ti run `npm run format:check`, what in my terminal is look like the image I attached. Then I run `npx prettier --write .`

Reply:
It all passes now